### PR TITLE
Modified Pie to allow user to set a Percent Value Radius

### PR DESCRIPTION
**Pull request by louisreynaud**
Couldn't test it due to lack of time on my part. Maybe could you check it. Otherwise, no big deal, I'll work with the actual version.

Thanks if you check my work and add it to the actual and for your time.

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (szymach) [inline .vscode/launch.json]: This is for VisualStudioCode, yes? If so, please remove it, there should be no IDE/editor specific code in the library.

**Comment by EkinOf** [comment]
@szymach  Need this feature too, can you accept it please ?

**Comment by szymach** [comment]
Hey, so I ran the test from your branch and these are the result images I have got:

draw2dpie
draw2dring
draw3dpie
draw3dring

As you can see, either you have made a mistake somewhere in the code or the tests need adjustments. Should you wish to see the generated images. comment out [this part](https://github.com/szymach/c-pchart/blob/master/tests/_support/Helper/Unit.php#L26) of the code and the files will be in `tests/_output/charts`

**Comment by louisreynaud** [comment]
Hi, I'm experimenting troubles with travis CI when commiting. Actually I've just reviewed the tests so I think that the 3rd image of the tests you ran, the 3d Pies, will render fine now. On the other hand, the second one, with lots of labels stacking themselves, is a pure mystery to me. Last time I checked, I've not touched lines of code in relation with labels. 

Unfortunately I won't have much time to try to resolve this bug, as I'm working on another project. 

Right now, it seems to work for 2D Pies and 3D Pies, the percent is also working on the 2D Ring but seems to duplicate labels, even if I don't understand why, Finally, I've not set the functionality for 3D Ring due to the way it works which doesn't seems to allow the modification I made for the 3 others.

If you have the time to put an eye on it someday, it would be wonderful, until then I'll just use the actual version of c-pchart which is working for me. Sorry to bother you with this. I'd understand if you choose to close this pull request.

Thanks for your time.

**Comment by louisreynaud** [comment]
Thanks for the fast answer.
Yes that's the purpose of my changes. 

I wanted to allow the user to set the scale of the values from the center of the chart. Apparently the modification does what I wanted but seems to add a strange thing to the labels.

Anyway, thanks again

**Comment by szymach** [comment]
Closing as unresolved.

**Pull request closed without merging**